app/components/school_information/your_schools/navigation_component.html.erb

```diff
@@ -0,0 +1,11 @@
+<nav class="hmcts-sub-navigation" aria-label="Sub navigation">
+  <ul class="hmcts-sub-navigation__list">
+    <% @cohorts.each do |cohort| %>
+      <li class="hmcts-sub-navigation__item">
+        <%= govuk_link_to lead_providers_your_schools_path(selected_cohort_id: cohort.id), class: "hmcts-sub-navigation__link govuk-link--no-visited-state", "aria-current" => (cohort == @selected_cohort ? 'page' : nil) do %>
+          <%= cohort.start_year %> cohort
+        <% end %>
+      </li>
+    <% end %>
+  </ul>
+</nav>
```

Comment:
Is this component doing the same thing that `tabs` component that Callum added does? With tiny style variation? Should we use that one and maybe change the styles a little?

Reply:
Just made changes - modified the tabs component so it looks the same. It does look slightly messier though but happy with deleting code